core/tracepoint.c

```diff
@@ -61,7 +61,7 @@ int register_user_trace_event(void* self, char* description) {
 }
 
 void call_tracepoint(int event_type, void* reactor, tag_t tag, int worker, int src_id, int dst_id,
-                     instant_t* physical_time, trigger_t* trigger, interval_t extra_delay, bool is_interval_start) {
+                     instant_t* physical_time, trigger_t* trigger, interval_t extra_delay) {
```

Comment:
I'm not sure about this change. I thought the `is_interval_start` field was used to be able to determine the execution time of a reaction.  Are you sure we can still do that with it removed?  The thing to check is whether trace_to_chrome still displays the reaction executions as an interval.  I concur, however, that on searching for uses of this field, I can't find any, so this is probably right.

Reply:
Yeah, I recently removed the usage of that field as a simplification, since previously it was only used as an optimization to get extra measurement precision. We should add it back later when we have time and we realize we need it.

Reply:
We could keep it and add a `(void) is_interval_start` to avoid the warning